piperider_cli/event/__init__.py

```diff
@@ -127,3 +129,12 @@ def log_event(prop, event_type, **kwargs):
         **prop,
     )
     _collector.log_event(payload, event_type)
+
+
+class UserProfileConnector(object):
+    @staticmethod
+    def update(key: str, val: Any, name: str = None):
```

Comment:
could we allow the current `val` type annotation to be `Union[str, bool]`?

Reply:
fixed